packages/overlays/src/DropdownConfig.js

```diff
@@ -0,0 +1,17 @@
+export const DropdownConfig = () => ({
+  isGlobal: false,
+  hidesOnOutsideClick: true,
+  hidesOnEsc: true,
+  popperConfig: {
+    placement: 'bottom',
+    inheritsReferenceObjectWidth: true,
```

Comment:
New prop should go one level up. Also see general discussion. I think this object should only contain original Popper config as stated in their docs.